frontend/test/metabase/visualizations/lib/settings.unit.spec.js

```diff
@@ -172,6 +173,50 @@ describe("settings framework", () => {
       ]);
     });
   });
+
+  describe("mergeSettings", () => {
```

Comment:
@nemanjaglumac do you want these unit tests tagged with the issue? They were created while fixing a bug, but they're more specific and not exactly a repro like the Cypress test is.

Reply:
It doesn't hurt if we add a reference even if it's not a "pure" repro.